src/react-components/room/components/ChatToolbarButton/ChatToolbarButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { ToolbarButton } from "../../../input/ToolbarButton";
+// TO DO: look into changing icon theme handling to work with TS
+// @ts-ignore
+import { ReactComponent as ChatIcon } from "../../../icons/Chat.svg";
+import { FormattedMessage, defineMessage, useIntl } from "react-intl";
+import React, { RefAttributes, useContext } from "react";
+import { ChatContext } from "../../contexts/ChatContext";
+import { ToolTip } from "@mozilla/lilypad-ui";
+
+const chatTooltipDescription = defineMessage({
+  id: "chat-tooltip.description",
+  defaultMessage: "Open the chat sidebar (T)"
+});
+
+interface ChatToolbarButtonProps extends RefAttributes<any> {
```

Comment:
We usually opt in to using types before interfaces.  type ChatToolbarButtonPropsT = {}

This other suggestion is stylistic, we do it in the subscription team but we end interface,type, and enum with the first initial of what it is.  CategoriesE, ButtonsI ,ChatToolbarButtonPropsT etc.

I don't think extends RefAttributes<any> is being used here?